docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,107 @@
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "merged",
+      "attributes": {
+        "position": ["<your-gps-sensor-name-1>", "<your-gps-sensor-name-2>"],
+        "orientation" : ["<your-imu-sensor-name-1>"],
+        "compass_heading" : ["<your-gps-sensor-name-1>"],
+        "angular_velocity":["<your-imu-sensor-name-1>"],
+        "linear_velocity": ["<your-gps-sensor-name-1>"],
+        "linear_acceleration": ["<your-accelerometer-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":["vectornav", "mpu6050"],
+        "linear_velocity": ["gps1"],
+        "linear_acceleration": ["adxl345"]
+    }, 
+      "depends_on":[]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+Configure an array of the `name` of each movement sensor you want to add to your robot as a merged resource in the attributes of the `merged` movement sensor model:
+
+- The name of each attribute represents the `Property` that that particular movement sensor supports, or the type of reading or measurement that it takes.
+- Get the properties supported by each model from its [model configuration documentation](/components/movement-sensor/#configuration), or by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
+- Use this information to determine which attribute to put its `name` inside the array of.
+You can use the same sensor for multiple attributes if it supports multiple properties.
+
+Name | Type | Inclusion | Description
+---- | ---- | --------- | -----------
+`position` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads position. |
+`orientation` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads orientation. |
+`compass_heading` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads compass heading position. |
+`linear_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear velocity. |
+`angular_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads angular velocity. |
+`linear_acceleration` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear acceleration |
+
+Note that only one sensor from each array can be used to retrieve each type of reading.
```

Comment:
[opt] Could make this into an "Important" or "Tip" alert if you think it could be easily missed.

Reply:
I considered this as well but Rand didn't think it should be tip and I think it's slightly intuitive so probably ok